tools/manifest/item.py

```diff
@@ -52,12 +37,7 @@ def id(self):
 
     @property
     def meta_flags(self):
-        return set(self.source_file.meta_flags)
-
-    @property
-    def path(self):
-        """The test path relative to the test_root"""
-        return self.source_file.rel_path
+        return set()
```

Comment:
Is the belief that this isn't used anywhere? If so can we add a warning or something if we don't want to remove it?

Reply:
Yes. AFAICT, it is literally only used by `ManifestItem.https` and nowhere else, and we don't need to deal with that for cases where we don't have a URL. I guess `https` should really move to `URLManifestItem`, and `meta_flags` made private?